src/Instrumentation/MySqli/src/MySqliInstrumentation.php

```diff
@@ -466,6 +512,14 @@ private static function queryPostHook(CachedInstrumentation $instrumentation, My
 
     }
 
+    /** @param non-empty-string $spanName */
+    private static function multiQueryPreHook(string $spanName, CachedInstrumentation $instrumentation, MySqliTracker $tracker, $obj, array $params, ?string $class, string $function, ?string $filename, ?int $lineno): void
+    {
+        $span = self::startSpan($spanName, $instrumentation, $class, $function, $filename, $lineno, []);
+        $mysqli = $obj ? $obj : $params[0];
+        self::addTransactionLink($tracker, $span, $mysqli);
+    }
```

Comment:
Just curious why this prehook was added?

Reply:
It is for `multi_query` scenario where it works so differently when compared to `query` and `real_query`.

Reply:
Still not seeing what this added code is doing for multi query, can you explain.